### DISCUSSION TITLE
Chocolatey packages for GLPI Agent

### DISCUSSION THREAD

**Discussion by C-Duv** [glpi-project/glpi-agent#428] [Show and tell]
I recently published a [Chocolatey](https://www.chocolatey.org) package for the nightly builds of the GLPI Agent : [`glpi-agent-nightly`](https://community.chocolatey.org/packages/glpi-agent-nightly)

*Ayt3ck* has also published a Chocolatey package for the stable releases of the GLPI Agent : [`glpi-agent`](https://community.chocolatey.org/packages/glpi-agent)

For those not aware of Chocolatey, it's way to install software on Windows using command line (e.g. `choco install glpi-agent-nightly`).

Maybe this alternative way of installing the  agent (stable or nightly) could be added somewhere in the documentation?

I personally uses this at my current job (and planning to use Ansible role for simple deployment) where switching from [FusionInventory Agent (also available as a Chocolatey package)](https://community.chocolatey.org/packages/fusioninventory-agent) to GLPI Agent was done using only 2 CLI commands.

**Comment by g-bougard** [comment]
Hello @C-Duv 

thank you for your suggestion.

As you should know the documentation is also free and sources are available on github here: https://github.com/glpi-project/doc-agent

I can propose you to as some dedicated documentation text here: https://github.com/glpi-project/doc-agent/blob/3cc0cc126a570723d912cdd646c2a9179a3cfab8/source/installation/index.rst#L39
So it would appear just after "Large Installation" and just before "Contributions".
You can of course fork the documentation project and submit a PR I would be happy to review & merge.
You can even want to add a dedicated page like the one I did for "Command line parameters" with some usage examples or a full tuto. This is up to you.

About the text to add I would appreciate you provide it as you are still using this solution. Personally I still used Chocolatey for other usage (including Strawberry Perl) so I know this can be useful.

**Reply by C-Duv** [reply]
Thanks, I'll submit a PR to add it to the doc at the suggested place.

Depending on how long my text will be I might create a dedicated page: we'll see.
